lib/game.rb

```diff
@@ -0,0 +1,71 @@
+class Game
+  attr_reader :current_turn, :turn_count, :turn_return
+  def initialize(current_turn)
+    @current_turn = current_turn
+    @turn_count = 1
+    @turn_return = nil
```

Comment:
Just reading this class from top to bottom, I'm not exactly sure what `@turn_return` is right now? What would have been a better, more descriptive name to use?